Cargo.toml

```diff
@@ -40,8 +41,10 @@ optional = true
 version = "0.3"
 
 [features]
-default = ["tls"]
+default = ["use_dirs", "tls"]
```

Comment:
Sorry for the confusion. I meant to only depend on it when the httpcache flag is enabled. That's the only place I think it's actually used. I don't think we need a separate. Flag for this just

httpcache = ["dirs"]

:)

Reply:
Oh right!  Sure.